pywbem/cim_operations.py

```diff
@@ -587,6 +590,25 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
             Initial enablement status for maintaining statistics about the
             WBEM operations executed via this connection. See the
             :ref:`Statistics` section for details.
+
+          enable_logging (:term:`string` or None):
+            *New in pywbem 0.12 as experimental
+            Enable logging of operation requests and/or xml requests and
+            responses and set the detail level for the logging.
+
+            * If None logging is not enabled.
+
+            * If 'min' the logging is set to capture to a maximum log size
+              defined by the `DEFAULT_MAX_LOG_ENTRY_SIZE` variable.
+
+            * If 'all' the full data content is captured.
```

Comment:
The description of the three list items above suggests that they enable or disable the logging. That is not the entire truth, because it also depends on the log levels of the respective Python logger objects. I know that the following paragraph is supposed to state that, but I find it confusing, and would like to have some more precision. For example, both these items and the following paragraph talk about "capturing", without detailing what that means. The fact is that the enabled_logging parameter controls a basic enablement that is a precondition for the usual controls on the logger objects. Maybe we should explain it here with this notion of a prerequisite, and otherwise point to the logging section as the text already does.

Reply:
Think I have improved it but I want to read it all over again and compare with the doc in _logging.